docs/SSH.md

```diff
@@ -1,47 +1,83 @@
 
-## Enable SSH
+Here are a few (free) popular clients to try. These need to be installed on the PC, Mac, or other computer you are using to connect *to* the Raspberry Pi:
 
-**Note:** Starting with RetroPie 4.2, in order to keep the default image secure, SSH is disabled by default. It can be enabled in either of the two ways listed below.
+* Windows:
+  * [Putty](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html) - Very simplistic access to allow for console commands, but does not feature the ease of drag & drop for ROMs and other files.
+  * [WinSCP](https://winscp.net/eng/download.php) - An SFTP client that features an easy way to drag & drop files. Console commands are also possible (and even integrates with Putty) but is a secondary user interface found in the `Commands` > `Open Terminal` or `Commands` > `Open in Putty` menus.
+  * [MobaXterm](https://mobaxterm.mobatek.net/) - A feature-rich console commands and drag & drop client that includes most ways to remotely connect to another computer, including SSH, SFTP, and even VNC (allows viewing the screen of another computer).
+* Mac:
+  * [CyberDuck](https://cyberduck.io/?l=en)
 
-1) Pre-boot. On a system with an SD-card reader, access your SD-card and create an empty file called `ssh` or `ssh.txt` in the root directory of the **boot** partition.
 
-2) In **raspi-config**: after booting.
-```
-sudo raspi-config
-```
-**interfacing options >> SSH >> Enable >> reboot your pi**
+# Enable SSH
+
+> **NOTE:** 
+>
+> Starting with RetroPie 4.2, in order to keep the default image secure, [SSH is disabled by default](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/). You will not be able to remotely connect to it until it has been enabled using **ONE** of the instructions below.
+
+## (Option 1) From the SD Card
+
+If your computer has an SD-card reader or a special USB stick that allows inserting an SD card, plug it into your computer, open the new drive to access your SD-card's `boot` partition,  and create an empty file called `ssh` or `ssh.txt` in the root directory. Ignore any warnings about the drive needing to be repaired.
+
+## (Option 2) From the RetroPie menu
 
-For more information on why SSH was disabled see [HERE](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/).
+1. Select `raspi-config` from the RetroPie menu after booting up.
+2. Select `Interface Options`
+3. Select `SSH`
+4. Choose `Yes`
+5. Select `Ok`
+6. Choose `Finish`
 
-### When using BerryBoot
+## (Option 3) When using BerryBoot
 
 When using RetroPie with [BerryBoot](http://www.berryterminal.com/doku.php/berryboot) you cannot use `raspi-config`. There is a workaround to enable SSH:
-* Boot to RetroPie (via BerryBoot)
-* Enter shell by pressing <kbd>Ctrl + F4</kbd>
-* `cd /boot`
-* `sudo touch ssh`
-* `sudo reboot now`
-* Done!
+1. Boot to RetroPie (via BerryBoot)
```

Comment:
The list needs a space before, otherwise MkDocs doesn't format it properly.

Reply:
Another thing not shown in the preview? Done.